docs/pages/admin-guides/access-controls/idps/saml-microsoft-entra-external-id.mdx

```diff
@@ -0,0 +1,235 @@
+the `DirectFedAuthUrl` value of the `TXT` record points to SAML IdP SSO URL of Teleport cluster `mytenant.teleport.sh`.  
+```
+example.com.  IN   TXT   DirectFedAuthUrl=https://mytenant.teleport.sh/enterprise/saml-idp/sso
+```
+
+For the self-hosted Teleport cluster, determine if DNS update is required based on the reference table shown above. 
+
+For the Teleport Cloud cluster, you must update the DNS record of your domain. Because your Teleport cluster is 
+always hosted under `mytenant.teleport.sh` domain and your user account will not belong to the `teleport.sh` domain.
+
+## Step 2/5. Configure Teleport as an identity provider
+
+In this step, we configure Teleport SAML IdP as an identity provider in Microsoft Entra External ID
+and add Microsoft Entra External ID as a service provider in Teleport. Teleport Web UI offers a guided 
+flow to configure end-to-end SAML integration.
+
+From the Teleport Web UI, locate **Add New > Resource** menu. From the list of resource catalog
+select **Microsoft Entra External ID** tile. 
+
+![Add new resource](../../../../img/access-controls/saml-idp/ms-entra-external-id/add-new-resource.png)
+
+Once you click on the tile, the Web UI guides you through steps of adding 
+Teleport as an identity provider in Microsoft Entra External ID.
+
+You can locate **Microsoft Entra External ID** in Azure portal under Azure services.
+
+![Add new SAML IdP in Microsoft Entra External ID](../../../../img/access-controls/saml-idp/ms-entra-external-id/entra-add-idp.png)
+
+Note that while adding Teleport as an identity provider, you need to provide domain name 
+as explained in the Step 1 of this guide.
+
+Once you add Teleport as SAML IdP in Microsoft Entra External ID, you will need to copy the Entra ID `Tenant ID` from
+Azure portal and provide the value to the Web UI and move to the next step of the SAML service provider configuration. 
+
+As a second step of the SAML service provider configuration, you will notice that the Web UI already configures 
+the SAML service provider entity ID, ACS URL and default attribute mapping value that is required for the integration. 
+These values are generated based on preset values and Entra ID `Tenant ID`.
+
+For reference, the values that are auto-configured are listed below:
+
+- Entity ID: `https://login.microsoftonline.com/<entra-id-tenant-id>/`
+- ACS URL: `https://login.microsoftonline.com/login.srf`
+- Attribute:
+  - Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress`
+  - Value: `uid`
+- Launch URL: `https://portal.azure.com/<entra-id-tenant-id>`. The launch URL is where users are sent to after
+a successful authentication with Teleport.
+
+Provide a display name for this integration (e.g. "azure_portal") and click on button named 
+`Finish` to register service provider. 
+
+## Step 3/5. Create and invite user
+
+In this step, we will create and invite an external user that authenticates with Teleport SAML IdP.
```

Comment:
```suggestion
In this step, we will create and invite an external user that authenticates with the Teleport SAML IdP.
```
Since this is a common noun and not a feature/product name.